modules/js/index.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-restricted-globals */
+/*  eslint linebreak-style: ["error", "unix"]   */
+/* eslint-disable no-undef */
```

Comment:
- [ ] I suggest that you disable these highlighted linters because they help us as developers to write code that is up to standards. It would be nice if you just fixed the code they complain about 😄.